data/.wrglsync.yaml

```diff
@@ -576,6 +584,10 @@ repos:
     fileName: fuse/event_terrebonne_so.csv
     primaryKey:
       - event_uid
+  event-acadia-so:
+    fileName: fuse/event_acadia_so.csv
+    primaryKey:
+      - event_uid
```

Comment:
Please merge the `main` branch into this branch and modify the `.wrgl/config.yaml` file instead.

Reply:
Got it. Thanks.